apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/Program.cs

```diff
@@ -64,14 +77,16 @@ internal class Program : YcsbRunner
                 {
                     await Task.Yield();
                     var sw = Stopwatch.StartNew();
-                    var cmd = s_connection.CreateSelectCommand(
-                        s_selectQuery, new SpannerParameterCollection
-                        {
-                            {"p", SpannerDbType.String, s_keys[Rand.Value.Next(s_keys.Count)]}
-                        });
-                    using (var reader = await cmd.ExecuteReaderAsync(s_timestampbound).ConfigureAwait(false))
+                    if (!IsEmpty)
```

Comment:
It feels like having separate classes here (one for raw, one for empty, one for regular) might make a lot of this code a lot simpler. I'm happy to treat all of this as slightly experimental though, and we can refactor later, when we have a better idea of what features we really need.